app/rest/docker/taskLogs.js

```diff
@@ -0,0 +1,18 @@
+angular.module('portainer.rest')
+.factory('TaskLogs', ['$http', 'DOCKER_ENDPOINT', 'EndpointProvider', function ServiceLogsFactory($http, DOCKER_ENDPOINT, EndpointProvider) {
+  'use strict';
+  return {
+    get: function (id, params, callback) {
+      $http({
+        method: 'GET',
+        url: DOCKER_ENDPOINT + '/' + EndpointProvider.endpointID() + '/tasks/' + id + '/logs',
+        params: {
+          'stdout': params.stdout || 0,
+          'stderr': params.stderr || 0,
+          'timestamps': params.timestamps || 0,
+          'tail': params.tail || 'all'
+        }
+      }).success(callback).error(function (data, status, headers, config) {});
```

Comment:
Every factory in the `portainer.rest` module use `$resource` instead of `$http` except the followings:

* ContainerLogs
* ServiceLogs
* TaskLogs

In these 3, the request failure is ignored at the moment. IMO they should all be refactored to use `$resource` and promises so that the success/error handling is done in services instead of here.

I think that we should allow this to be merged and refactor the 3 components inside the same commit.